index.js

```diff
@@ -1,257 +1,245 @@
-        description: `An in-depth look at Hollywood's depiction of transgender people and the impact of those stories on transgender lives and American culture.`,
-        imageUrl: 'https://flxt.tmsimg.com/assets/p18415918_p_v13_aa.jpg'
-    },
-    {
-        title: 'Passing',
-        director: [{"name":'Rebecca Hall', "year of birth":'1982',"year of death":'', "bio":'Rebecca Maria Hall is an English actress and filmmaker from London.She made her first onscreen appearance at age 10 in the 1992 television adaptation of The Camomile Lawn, directed by her father, Sir Peter Hall. In 2006, following her film debut in \'Starter for 10\', Hall got her breakthrough role in Christopher Nolan\'s thriller film \'The Prestige\'. In 2008, she starred as Vicky in Woody Allen\'s romantic comedy-drama \'Vicky Cristina Barcelona\', for which she received a Golden Globe nomination for Best Actress. Hall then appeared in a wide array of films.She made her directorial debut with Passing (2021), receiving critical acclaim.'}],
-        year: 2021,
-        genre: [{name:'Drama', description:'The drama genre features stories with high stakes and many conflicts. They’re plot-driven and demand that every character and scene move the story forward. Dramas follow a clearly defined narrative plot structure, portraying real-life scenarios or extreme situations with emotionally-driven characters. Films that fall into drama sub-genres include historical drama or costume drama, romantic drama, teen drama, medical drama, docudrama, film noir, and neo-noir.'}],
-        description: `In 1920s New York City, a Black woman finds her world upended when her life becomes intertwined with a former childhood friend who's passing as white.`,
-        imageUrl: 'https://flxt.tmsimg.com/assets/p20588150_p_v10_aa.jpg'
-    }
-]
+};
+
 
 app.use(morgan('common'));
 
 app.use(express.static('public',options));
 
 // GET requests
 app.get('/',(req,res)=>{
-    res.send("<h1>This is just a default textual response. Nothing to worry about.</h1>")
+    res.send("<h1>This is just a default textual response. Nothing to worry about.</h1>");
 })
 
 // get all movies
 app.get('/movies',(req,res)=>{
-    res.json(movies)
+    //res.json(movies)
+    Movies.find()
+        .then((movies) => {
+            res.status(201).json(movies);
+        })
+        .catch((error)=>{
+            console.error(error);
+            res.status(500).send('Error ' + err);
+        })
+});
+
+// get all genres
+app.get('/genres',(req,res)=>{
+    Genres.find()
+        .then((genres) => {
+            res.status(201).json(genres);
```

Comment:
Response of `201` means created, let us check how to properly use [Http response status code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)